common/gossmap.c

```diff
@@ -436,6 +436,13 @@ static struct gossmap_chan *add_channel(struct gossmap *map,
 	return chan;
 }
 
+static u32 peek_cupdate_timestamp(struct gossmap *map, size_t cupdate_off){
```

Comment:
Pretty sure this commit should be omitted, once we don't allow timestamps to go backwards any more.